src/SonarScanner.MSBuild.Tasks/GetAnalyzerSettings.cs

```diff
@@ -286,6 +296,17 @@ private static void WriteMergedRuleSet(string originalRuleset, string languageRu
 
         private AnalyzerSettings GetLanguageSpecificSettings(AnalysisConfig config)
         {
+            if (config == null)
+            {
+                return null;
+            }
+
+            if (string.IsNullOrEmpty(Language))
+            {
+                Log.LogMessage(Resources.AnalyzerSettings_LanguageNotSpecified);
+                return null;
+            }
+            
```

Comment:
These checks were being done elsewhere before. This method is now called first by the task, so I've moved them here. This also preserves the previous behaviour.